### PR TITLE
pacs.008.001.10

### PR DESCRIPTION
**Pull request by vorsterk**
Committing transfer service changes for pacs.008.001.10 for the TMS service

### CODE REVIEW

File: src/router.ts [hunk 1]
```diff
@@ -8,10 +8,12 @@ const router = new Router();
...
-// execute the service
+// execute the service Pain001
 router.post('/execute', monitorQuote);
+// ReplyQuote Pain013
 router.post('/quoteReply', replyQuote);
-// router.post('/transfer', monitorTransfer);
+// Transfer Pacs008
+router.post('/transfer', monitorTransfer);
 
 // transfer response
```
Comment (theolextego) [inline src/router.ts:18]: Just add Pacs002 here also

**Review by theolextego — APPROVED** [review]
Good one Kyle 👌🏽

**Review by uche-lextego — APPROVED** [review]
LGTM. Kyle 💯

**Review by uche-lextego — APPROVED** [review]
LGTM

**Pull request merged**